### ISSUE TITLE
is there a slidesToShow option available 

### ISSUE THREAD

**Issue by holligans** [xiaolin/react-image-gallery#323]
Is there any prop that allows me to display multiples slides like "slidesToShow: 3"

**Comment by xiaolin** [comment]
No, but you should be able to achieve that via overriding CSS. if you resize each slide to about `33.3%`

**Comment by holligans** [comment]
Thanks for the quick response, I had tried your approach before and It worked perfectly but lazyLoad and other features doesn't work correctly, maybe I'm missing something.

**Comment by xiaolin** [comment]
Right, its a workaround and not officially supported to show more than one slide at a time, so props like `lazyLoad` and other features may not work correctly.

**Comment by yasserzubair** [comment]
has anyone achieved this? @holligans @xiaolin

**Comment by holligans** [comment]
Nope
